tools/codegen-gradle-plugin/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
     compile project(':languages:typescript')
     compile project(':languages:postman')
     compile project(':languages:python')
-    compile project(':languages:php')
+    compile project(':languages:go')
```

Comment:
please don't remove the php project, even if you dislike the language ;)

Reply:
Haha oops, sorry :-) Making some progress on the go sdk, trying to finish things up. Didn't realize this was still visibile here :-)